ecosystem/sep-0009.md

```diff
@@ -28,6 +28,27 @@ date | 10 | [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) date-only format
 phone number | varies | [E.164](https://en.wikipedia.org/wiki/E.164)
 occupation | 3 | [ISCO08](https://en.wikipedia.org/wiki/International_Standard_Classification_of_Occupations)
 
+## Field Naming Conventions
+
+Where possible we use field names from [schema.org](https://schema.org/Person). Words are separated with underlines as that convention has previously been established in Stellar protocols.
+
+### Dot Notation
+
+Field names should always be used as strings. For example:
+```json
+{
+	"organization.name": "Stellar Development Foundation"
+}
+```
+The dot notation is _not_ an indication that the fields described should be contained in a nested object under a top-level key. For example, the following would not be compliant:
+```json
+{
+	"organization": {
+		"name": "Stellar Development Foundation"
+	}
+}
+```
```

Comment:
💡 I think stating what it shouldn't look like might backfire if someone quickly reading the spec thinks this is an example. But if you think it is necessary to prevent mistakes or assumptions, looks good.